src/v4.js

```diff
@@ -2,10 +2,10 @@ import rng from './rng.js';
 import bytesToUuid from './bytesToUuid.js';
 
 function v4(options, buf, offset) {
-  const i = (buf && offset) || 0;
+  const start = (buf && offset) || 0;
 
   if (typeof options === 'string') {
```

Comment:
It would be nice to finally get rid of this legacy old API:

```js
uuidv4('binary');
```

Out of scope for this PR, but I've taken note in #437.

Reply:
@ctavan ok, I will do it a bit later (deprecated)